CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+### How to make a clean Pull Request:
+
+1. Make sure your pull request has a correct title. For example in PlatformUI we use 'EZP-XXXXX: Title of the pull request'
```

Comment:
You are right. So a part explaining how to format your commit should be added. Maybe also explain how they should be split.